syncano_cli/main.py

```diff
@@ -67,14 +77,14 @@ def login(args):
         print(error)
 
 
-@command
-@argument('-s', '--script', action='append', dest='scripts',
+@Command('sync')
```

Comment:
Class based decorators don't work on methods http://stackoverflow.com/questions/6676015/class-decorators-vs-function-decorators . You could just provide an argument for command decorator to define if it's high-level command or subcommand. I see no reason for such a rewrite, but let it stay this way.